integration/kubernetes/README.md

```diff
@@ -0,0 +1,46 @@
+## Pre-requisites
+
+Alluxio workers use `emptyDir` volumes with `sizeLimit`; an alpha feature in Kubernetes 1.8. Please ensure the feature is enabled.
```

Comment:
the general rule for semicolons is that each side should be a complete sentence, but "an alpha feature in Kubernetes 1.8" isn't a complete sentence